certified-connectors/UiPath Orchestrator/apiProperties.json

```diff
@@ -29,15 +29,15 @@
           {
             "authorizationUrl":
             {
-              "value": "https://cloud.uipath.com/identity_/connect/authorize"
+              "value": "https://staging.uipath.com/identity_/connect/authorize"
```

Comment:
@TuckermanRav could you please explain the usage of staging environment here? Is there any specific reason for it? It's not allowed to use test or staging environments.

Reply:
Our original thought was that during the Microsoft "review/approval" process we would use our staging environment. However, based on your feedback we've decided to go ahead and use production.  Thanks. I've reverted back to the cloud urls.

Reply:
@v-EgorKozhadei What we are trying to do is test this feature across in an integrated manner. Since we have timeline on Prod Deployments, we want to make sure all tests have passed from MS before we change the urls. We will submit another PR that will have updated Urls that will point to our Production Environments.